.prettierignore

```diff
@@ -7,3 +7,4 @@ app/vendor/**
 .nyc_output/**
 .vscode/**
 test/e2e/send-eth-with-private-key-test/**
+*.scss
```

Comment:
... was `prettier`  linting our scss before now? :thinking:

Reply:
it was... and I think we should re-enable it once we tweak some stylelint settings?

It was breaking apart the src: lines in the typography file into multiple lines and stylelint would then complain about that.